com.kms.cura.entity/src/com/kms/cura/entity/user/UserEntity.java

```diff
@@ -29,5 +31,9 @@ public String getPassword() {
 	public void setPassword(String password) {
 		this.password = password;
 	}
+	
+	public static int getType() {
```

Comment:
make this method un-static and let DoctorUserEntity and PatientUserEntity override it